### ISSUE TITLE
SP-API Developer Profile Update : Restricted Access Request

### ISSUE THREAD

**Issue by EduardHov** [amzn/selling-partner-api-docs#2390]
Hi,

Amazon rejects our request of Developer Profile Update. During last 5 months we requested about 6-7 times and received the same message "We have reviewed the information you provided and determined that you will not be given access to Restricted SP-API roles based on your application. Your SP-API access remains unchanged at this time."  We want to include Direct-to-Consumer Shipping role to our profile.
Our customers used our application without any issue and complaint more than one year. One day Amazon decided to change our profile without any reason, which brings a big damage to our business and our customer's businesses.
Please help us to go forward with this problem.

**Comment by LindaNC** [comment]
Hello @EduardHov,

Thank you for bringing up the issue regarding the delay experienced with your {registration/appeal/PII Access request} process. So we can contact the appropriate team, please write back to us with the Registration Case ID.

Thanks,
Linda
Selling Partner API Developer Support

**Comment by WWangYu** [comment]
> Hello @EduardHov,
> 
> Thank you for bringing up the issue regarding the delay experienced with your {registration/appeal/PII Access request} process. So we can contact the appropriate team, please write back to us with the Registration Case ID.
> 
> Thanks, Linda Selling Partner API Developer Support

Thank you for your reply, I have reapplied, the id is: 10302865691

**Comment by EduardHov** [comment]
> Hello @EduardHov,
> 
> Thank you for bringing up the issue regarding the delay experienced with your {registration/appeal/PII Access request} process. So we can contact the appropriate team, please write back to us with the Registration Case ID.
> 
> Thanks, Linda Selling Partner API Developer Support

Hi @LindaNC,

Thanks for your attenation.
Case ID 10232095251.

Thanks,
Eduard

**Comment by EduardHov** [comment]
Hi @LindaNC,

Is there any news related to our problem?

Thanks,
Eduard

**Comment by AtikSrl** [comment]
hello @LindaNC 
We also have the same problem in the last 2 months. Amazon responds with a refusal to our every request for activation of developer credentials.

One of the reasons was "we only consider sharing Customer data (“Restricted access”) with third party Developers that offer features that are materially different from existing applications", statement of which we did not understand the meaning and how to possibly manage it.

our case: ID 7827078192

let me know if it is necessary for us to open a new issue about
thanks
Mauro

**Comment by jaimezac27** [comment]
hello @LindaNC 

I have the exactly same issue. The case ID is 10564003241. 

Can you help me? 

Let me know if you need any information

**Comment by hussain3134** [comment]
@EduardHov Did you get approved for restricted access?

**Comment by EduardHov** [comment]
> @EduardHov Did you get approved for restricted access?

Nop :((

**Comment by jaimezac27** [comment]
@hussain3134 not yet

**Comment by jaimezac27** [comment]
I got an answer, but they dont they the problem, how am I gonna fix it?

They replied:
"For Restricted roles:

 Additionally, we have also determined that you will not be given access to Restricted SP-API roles based on your application. Your SP-API access remains unchanged at this time.

We evaluate all Public Developer applications that request Restricted SP-API access, for a variety of technical and business criteria including, but not limited to, launch readiness, services offered that can be viewed on a public facing website, and supported geographies."

**Comment by quocvu88** [comment]
Hello @LindaNC 
I have the same issue. Please help me. CaseID:10597629541.
Thank you!

**Comment by pazion** [comment]
We have the same problem, denied the the restricted data token.
We help customers automate the orderprocessing, but this way Amazon is a no-option.

**Comment by EduardHov** [comment]
We changed all functionality of our Amazon Connector. Fully. And satisficed all points to Amazon requirements, but they rejected again :((( I don't know what to do again...

**Comment by Mahesh-stockone** [comment]
Hello @LindaNC 
I have submitted a request for PII but i have got no response for it.
Please review my case and update me.
my case id 8123630402
Thank You

**Comment by Olijve-CV** [comment]
Hello @LindaNC
I have submitted a request for PII but i have got no response for it.
Please review my case and update me.
my case id 11102462441
Thank You

**Comment by ravidGoldenberg** [comment]
Hi @LindaNC @jchavest13 I see there is still a lot of developers stuck on this issues, can anyone help making this process more transparent so developers  could understand what they actually need to change/fix to be granted PII?  
We are also stuck on this matter, case id 12297523981, please review our case and update us on what is the reason we have been denied PII

**Comment by umut77** [comment]
We are having the same issue as well, providing e-commerce  logistics service to Etsy , amazon , Walmart and eBay registered 40000+ customers with their shipments globally and still getting refused, seriously what other options/ways to apply this??

**Comment by ericahoekerd** [comment]
Our submission is also rejected (for several times now over the last 2 years), the last reason they gave:

_We have completed our assessment and have determined that you do not meet the requirements for Restricted SP-API. To protect Amazon Customers, we only consider sharing Customer data (“Restricted access”) with third party Developers that offer features that are materially different from existing applications. The Developers that meet these criteria must then go through rigorous security reviews with Amazon._

How can an integration between Amazon and Dynamics 365 differs from other integration applications? API's are used for creating seamless integrations. Anyone at Amazon who can shine a light why it is so restricted to get address information in an order, so the merchant knows where to ship the order?

**Comment by NA-Dev** [comment]
My company's submission was also rejected for the Restricted SP-API role. They do not make it clear at all what the requirements are to be approved. Why can a large 1P seller on Amazon not gain access to their own data?

**Comment by Iron27** [comment]
We've answered properly in detail three rounds now and they provide ZERO helpful information about what exactly was wrong in our responses.

Amazon is a corrupt monopoly.

**Comment by janhellopine** [comment]
@Iron27 were you able to get the restricted access after all?

**Comment by Iron27** [comment]
> @Iron27 were you able to get the restricted access after all?

Nope.  We quit trying.  Importing into our system via a CSV from Seller Dashboard instead.  Several more steps every day, but appealing the API access with Amazon's robots is a waste of time.

**Comment by danialpato** [comment]
Our company has integrated with many online stores in Türkiye. As Trendiol, Ozon, HepsiBurada, Etsy, etc., none objected to this access, but Amazon has destroyed our project. Who can help us? Our customers need this system, but Amazon does not cooperate and finally sends us the following message.

"We have completed our assessment and have determined that you do not meet the requirements for Restricted SP-API. To protect Amazon Customers, we only consider sharing Customer data (“Restricted access”) with third-party Developers that offer features that are materially different from existing applications. The Developers that meet these criteria must then go through rigorous security reviews with Amazon.

No changes have been made to your SP-API access at this time.
"
Case Id : "9529826412"

@LindaNC
